server/routes/inPrisonEducationRecordsResolver.ts

```diff
@@ -18,4 +18,19 @@ const completedInPrisonEducationRecords = (
   }
 }
 
-export default completedInPrisonEducationRecords
+/**
+ * Returns an InPrisonEducationRecords object where the eduction records are filtered such that only the n most recent
+ * completed courses are returned; where n is the parameter `numberOfRecordsToReturn`. The default if not specified is 2.
+ */
+const mostRecentCompletedInPrisonEducationRecords = (
```

Comment:
This function is to support the requirement on the Overview tab that we only display the 2 most recent Completed Qualifications.
Why 2 ?? Who knows? 🤷‍♂️ , but its one of the reasons I made the number a function parameter so when the business inevitably change their mind it should be pretty simple to do